### PR TITLE
Move tags to original text

### PR DESCRIPTION
**Pull request by gauravtiwari**
@sindrenm @mathiasbno Moved the tags to original text to address #2

### CODE REVIEW

File: app/helpers/spots_helper.rb [hunk 1]
```diff
@@ -10,4 +10,12 @@ def display_day(date)
       date.strftime("dd.mm.")
     end
   end
+
+  def render_with_hashtags(text)
+    sanitize(text.gsub(/(?:#(\w+))/) {hashtag_link($1)})
```
Comment (sindrenm) [inline app/helpers/spots_helper.rb]: Is `sanitize` by default gonna leave the link there? (Just asking, cause I'm too lazy to check.)

**Comment by sindrenm** [comment]
I like this change, it fixes #2, as you say. :smile:

File: app/helpers/spots_helper.rb (hunk 1 above)
Comment (gauravtiwari) [inline app/helpers/spots_helper.rb]: @sindrenm  yeah, it will :)

**Comment by gauravtiwari** [comment]
@sindrenm Going to merge this then :hand: ?

**Comment by jgorset** [comment]
I'll do it!

**Comment by gauravtiwari** [comment]
:+1:

**Comment by sindrenm** [comment]
Yah, I was going to after you answered https://github.com/hyperoslo/spot-hyper/pull/4#discussion_r49319738. But thanks, @jgorset. :smile:

**Pull request merged**